d3a_api_client/redis_aggregator.py

```diff
@@ -216,6 +224,12 @@ def calculate_grid_fee(self, start_market_or_device_name: str,
     def get_uuid_from_area_name(self, name):
         return get_uuid_from_area_name_in_tree_dict(self.area_name_uuid_mapping, name)
 
+    def get_name_from_area_uuid(self, asset_uuid):
+        for area_name, area_uuids in self.area_name_uuid_mapping.items():
+            for area_uuid in area_uuids:
+                if area_uuid == asset_uuid:
+                    return area_name
```

Comment:
Why not making it a util function to avoid repeatition [here](https://github.com/gridsingularity/d3a-api-client/pull/140/files#diff-2555ccae154ab4d35c55438d87c152a155d253febd6282fe67f75d7f62766d27R169-R174)